### DISCUSSION TITLE
[Feature Request] Add the indexer used per show to the shows details page.

### DISCUSSION THREAD

**Discussion by nydave69** [pymedusa/Medusa#9945] [Ideas & Feature requests]
Add the indexer used per show to the shows details page. since there is more than one now..

Maybe included in this list 
![image](https://user-images.githubusercontent.com/14140816/137339423-446aacdb-ca82-4717-bb1a-fb9331635706.png)

**Comment by medariox** [comment]
It's already there, marked with a star: 
![image](https://i.imgur.com/yKrVkDc.png)

**Reply by xbette** [reply]
The star is actually making it hard to press the ikon, you have to put the mouse a bit to the left for the star. Maybe the star can be removed, or even makes it click able like the icon?

**Reply by p0psicles** [reply]
Fixed in develop

**Comment by nydave69** [comment]
Ok...  learned something new..
